src/index.scss

```diff
@@ -0,0 +1,5 @@
+body {
+    div#root{
+      background-color: rgb(255, 255, 255);
+    }
+}
```

Comment:
@NLSanyu. I think it could be better if we could use eslint. For example we need a new line at the end of each file.What do u think?

Reply:
I agree. Let me do that

Reply:
@akram256 Done